packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/widgets.jsx

```diff
@@ -113,7 +112,13 @@ const RadioGroupOptionsWidget = ({
                     </span>
                 )}
                 {should_show_new_label && (
-                    <Text className='dc-dropdown__label--new' weight='bold' size='xxxs' line_height='s'>
+                    <Text
+                        className='dc-dropdown__label--new'
+                        weight='bold'
+                        size='xxxs'
+                        line_height='s'
+                        color='colored-background'
```

Comment:
what does colored-background do ?

Reply:
@henry-deriv i think it's being set into a css var: `var(--text-colored-background)`

Reply:
But it worked with light/dark mode even before putting that prop in before right?

Reply:
I decided to move text color inside prop of `<Text />` component and removed from CSS. Wanted not add style in CSS (like font-weight, font-size, color, line-height  and etc), but use the provided ability of component `<Text />`

Reply:
Yes, worked with light/dark as before. Will add to the qa check-list just to be sure
<img width="702" alt="Screenshot 2023-10-27 at 13 14 45" src="https://github.com/binary-com/deriv-app/assets/121025168/c855a527-295b-44f8-b3a3-0578977895ee">